pkg/server/ca/manager/manager.go

```diff
@@ -749,6 +437,47 @@ func (m *Manager) notifyBundleLoaded(ctx context.Context) error {
 	)
 }
 
+func (m *Manager) activateJWTKey() {
+	m.c.Log.WithFields(logrus.Fields{
+		telemetry.Slot:       m.currentJWTKey.id,
+		telemetry.IssuedAt:   m.currentJWTKey.issuedAt,
+		telemetry.Expiration: m.currentJWTKey.jwtKey.NotAfter,
+	}).Info("JWT key activated")
+	telemetry_server.IncrActivateJWTKeyManagerCounter(m.c.Metrics)
+	m.c.CA.SetJWTKey(m.currentJWTKey.jwtKey)
+}
+
+func (m *Manager) activateX509CA() {
+	m.c.Log.WithFields(logrus.Fields{
+		telemetry.Slot:       m.currentX509CA.id,
```

Comment:
This assumes that the caller of this function locks m.x509CAMutex. Would be problematic to have the lock here instead of in each caller so we don't inadvertently call this function without locking?

Reply:
that was my first approach `BUT` this function is used on Rotate that needs to add a mutex too, and result in a double lock...
So I choose to add locks on callers and call this from ActivateX509CA and in `RotateX509CA`

Reply:
I figured that could be the case.